.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.2
```

Comment:
If we want to enforce this, we should also update the jobs in the integrations to use this Ruby version.

Reply:
It seems like [the Semaphore build image would take care of that for us if `.ruby-version` is present](https://docs.semaphoreci.com/programming-languages/ruby/#selecting-a-ruby-version-on-linux), provided we don't force a different Ruby version.

The way it is worded in Semaphore's docs, however, makes me think that it'll only determine the Ruby version using the `.ruby-version` file if it's present _at the root of the repository_. On the other hand, they do say that it uses `rbenv`, and the `rbenv` docs say that [it should respect a `.ruby-version` file found _anywhere in the parent directories of the script that's being executed_](https://github.com/rbenv/rbenv#choosing-the-ruby-version). It's worth checking to see what it _actually_ does.

We do explicitly [set a different Ruby version for the diagnose tests in the Ruby integration](https://github.com/appsignal/appsignal-ruby/blob/main/.semaphore/semaphore.yml#L115-L116), although [it is not being _forced_](https://github.com/appsignal/appsignal-ruby/blob/main/.semaphore/semaphore.yml#L34-L36), so it might still ignore it and choose the version in `.ruby-version`. But that should fail, too, because of [the `support/check_versions` script](https://github.com/appsignal/appsignal-ruby/blob/main/support/check_versions) that is invoked after attempting to set the Ruby version. So I think the `RUBY_VERSION` environment variable should not be set for the Ruby integration diagnose tests, in order for it to work correctly. None of the other integrations seem to force a Ruby version.

Reply:
I'll merge this, and will start updating the three integrations afterward. Everything should work properly if Semaphore's `rbenv` works as expected.